### DISCUSSION TITLE
Can i change the default value of preferences parameter?

### DISCUSSION THREAD

**Discussion by Aayush04** [eclipse-theia/theia#7913] [General] [answered]
I am running my own version of theia. Using theia core 1.0.0
I wanted to change the default value of some of the preference parameters. And I wanted these changes to be done at the start of the theia from backend but not from setting.json file.
Is there any way to do it?

**Comment by akosyakov** [comment, marked as answer]
You can configure default preference values for a product:
https://github.com/eclipse-theia/theia/tree/master/dev-packages/cli#default-preferences

**Reply by Aayush04** [reply]
Thank you so much. I also got the same just now after searching a little.
